tests/Query/Functions/Sqlite/JsonGroupObject.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Scienta\DoctrineJsonFunctions\Tests\Query\Functions\Sqlite;
+
+use Scienta\DoctrineJsonFunctions\Tests\Query\SqliteTestCase;
+use Doctrine\ORM\Query\Expr;
+
+class JsonGroupObjectTest extends SqliteTestCase
```

Comment:
This file has a mismatch between the filename and the className. The file must be suffixed with `Test` in order to be runned in the test suite.